File: SecondYear/Anantj2499_AnantJaiswal_2226cseml1088_2/Week_3/SelectionSort.java
import java.util.* ;
import java.io.*;
public class SelectionSort {
    public static void selectionSort(int arr[], int n) {
        // Write your code here.
        int index =0;
        for(int i=0;i<n-1;i++){
            index =i;
            for(int j=i+1; j<n;j++){
                if(arr[j]<arr[index])
                    index =j;
            }
            if(index!=i){
                int temp = arr[i];
                arr[i]= arr[index];
                arr[index]= temp;
            }
        }
    }
}

File: SecondYear/aayushker_AayushkerSingh_2226CSE(AI)1010_2/Week_3/Selection_Sort.py
from os import *
from sys import *
from collections import *
from math import *

def selectionSort(arr,n):
    # Write your code here
    # Do not return anything. Update the given array in-place
    n = len(arr)

    for i in range(n):
        li = i
        
        for j in range(i + 1, n):
            if arr[j] < arr[li]:
                li = j

        arr[i], arr[li] = arr[li], arr[i]
    return arr
    pass